packages/app/src/systems/Account/hooks/useAccounts.tsx

```diff
@@ -79,6 +79,12 @@ export function useAccounts() {
     selectors.canHideAccounts
   );
 
+  function addAccount() {
+    store.send(Services.accounts, {
+      type: 'ADD_ACCOUNT',
+    });
+  }
```

Comment:
accountMachine is a global machine, for sending events to a global machine it's better to use `events.tsx` file inside `Accounts` system.